v4/weaviate/data/objectsGetter.go

```diff
@@ -55,6 +59,24 @@ func (getter *ObjectsGetter) WithLimit(limit int) *ObjectsGetter {
 	return getter
 }
 
+// WithConsistencyLevel determines how many replicas must acknowledge a request
+// before it is considered successful. Mutually exclusive with node_name param.
+// Can be one of 'ALL', 'ONE', or 'QUORUM'. Note that WithConsistencyLevel and
+// WithNodeName are mutually exclusive.
+func (getter *ObjectsGetter) WithConsistencyLevel(cl string) *ObjectsGetter {
```

Comment:
Should we add validation for the cl that is passed in? Or does Weaviate return an error message?

Reply:
Correct, this is validated sever-side